docs/recipes/SimplifiedGetCurrentBalanceFromAccount.md

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 1
+title: Simplified Get balance of the connected account
+description: Learn how to retrieve and display the ETH balance of the connected account in your dApp.
+---
+
+# Get the Current Balance of the Connected Account
+
+This recipe shows how to fetch and display the ETH balance of the currently connected account in your decentralized application. After the recipe code, you'll find an implementation guide that walks you through the steps of creating this recipe yourself.
+
+```tsx
+import type { NextPage } from "next";
```

Comment:
We are creating a component, so no need to import "NextPage"

Reply:
Fixed! 🙌